src/MongoDB.Analyzer/Core/Utilities/CollectionExtensions.cs

```diff
@@ -28,6 +41,19 @@ internal static class CollectionExtensions
     public static ImmutableArray<T> CreateImmutableArray<T>(params IEnumerable<T>[] enumerables) =>
         ImmutableArray.Create(enumerables.Aggregate((prev, curr) => prev.Concat(curr)).ToArray());
 
+    public static void RemoveRange<K, V>(this IDictionary<K, V> dictionary, IEnumerable<K> enumerable)
+    {
+        if(enumerable == null)
```

Comment:
Do a search in all code base for missing space in `if( `,` foreach(`, `for(` etc...